### PR TITLE
Added social media icons

### LINKED ISSUE #11: Add social icons for fellows
At the moment, it only displays the fellow name and location. To help people network, we should add social icons to each of the fellow cards.

Screenshot 2021-09-30 at 14 24 40
- [ ] Add LinkedIn icons to the card
- [ ] Add Twitter icons to the card
- [ ] Add Resume icons to the card
- [ ] Add GitHub icons to the card

### PR DESCRIPTION
**Pull request by prakharrathi25**
## Why this pull is important

At the moment, the cards only display the fellow name and location. To help people network, we have added social icons to each of the fellow cards. 

## What the pull request is adding
1. Added social media icons to the `descriptions.html` file. 
2. Added icons styling and position to the `all.scss` file 
3. Made edits to the `fellows.yml` file to allow for more links like LinkedIn, github, resume and twitter 
4. Add if conditions to check if the data exists 

## Screenshots 

image

## Checks

- [ ] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

## Contributors 

@prakharrathi25 
@arteevraina 

## Issues this closes 

This closes #11

### CODE REVIEW

**Comment by prakharrathi25** [comment]
@marcnjaramillo I couldn't reach you earlier so I am waiting for your approval before I can make the draft PR ready for review.

File: _data/fellows.yml
Comment (arteevraina) [inline _data/fellows.yml:4]: Maybe instead of Will's name. We should have all the names as Prakhar only. And in future, we can add the fellows name as well when we have complete data.

Comment (arteevraina) [reply _data/fellows.yml:4]: I am adding fix for this.

File: _data/fellows.yml
Comment (arteevraina) [inline _data/fellows.yml:10]: I don't think this is needed. Adding a fix for this as well.

File: _data/fellows.yml
Comment (prakharrathi25) [reply _data/fellows.yml:4]: That isn't required, actually. This is for testing purposes. Issue #13 is already working on that. Have discussed with @samarthya-jha and his team will add the data to the file.

File: _data/fellows.yml
Comment (prakharrathi25) [reply _data/fellows.yml:10]: Could you elaborate on why this isn't needed? And also please add what the high level fix is.

**Review by prakharrathi25 — COMMENTED** [review]
The images and the name can be left as it is since it is required for the header.

File: _data/fellows.yml
Comment (marcnjaramillo) [reply _data/fellows.yml:4]: @prakharrathi25 is correct. The scope of this issue is adding social media icons. This reduces any confusion or possibilities of merge conflicts.

File: _data/fellows.yml
Comment (marcnjaramillo) [reply _data/fellows.yml:10]: @arteevraina if you could add a little more detail in your reviews it would help us understand your thought process and have a more meaningful discussion.

**Review by marcnjaramillo — APPROVED** [review]
Good work both of you! I'll get this merged.

**Pull request merged**
